### PR TITLE
Make AppDesigner icon show outside of C#/VB

### PR DESCRIPTION
**Pull request by davkean**
This lets non-managed projects opt-into the AppDesigner folder and have the correct icon show.

Fixes: #2948.

### CODE REVIEW

File: src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Microsoft.VisualStudio.ProjectSystem.Managed.VS.csproj [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
 <Project Sdk="Microsoft.NET.Sdk">
-  <Import Project="..\VisualStudio.props"/>
+  <Import Project="..\VisualStudio.props" />
```
Comment (jmarolf) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Microsoft.VisualStudio.ProjectSystem.Managed.VS.csproj]: Can we remove unnecessary changes like this?

Comment (davkean) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Microsoft.VisualStudio.ProjectSystem.Managed.VS.csproj]: Hmm. Looks like we dirtied the project when I added the file - and that reformatted the doc. Given the current design of MSBuild (it goes through the XML API) - this isn't possible to avoid this change. Is that your question or was it "back out this change"?

Comment (jmarolf) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Microsoft.VisualStudio.ProjectSystem.Managed.VS.csproj]: I am fine with this as its going to happen anyway the next time someone edits this doc.  Just curious why it was edited at all as there were no changes made to the content.

Comment (davkean) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Microsoft.VisualStudio.ProjectSystem.Managed.VS.csproj]: Due to the way globbing works; CPS "dirties" the project to trigger a reevaluation. We have a bug on this.
